lib/active_merchant/billing/gateways/cyber_source.rb

```diff
@@ -746,6 +756,15 @@ def add_auth_service(xml, payment_method, options)
         end
       end
 
+      def add_threeds_exemption_data(xml, options)
+        exemption = options[:three_ds_exemption_type].to_sym
```

Comment:
If `options[:three_ds_exemption_type]` is nil, will calling `.to_sym` result in a error? Along the lines of "undefined method to_sym for nil:NilClass?"

Reply:
There's a check for a non-nil `options[:three_ds_exemption_type]` value before the method is called. I'll add an extra guard statement to that method to be on the safe side, though 👍🏾